pkgs/tools/X11/xpra/update.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnused common-updater-scripts
+
+testing=bullseye  # Current Debian testing distribution
```

Comment:
Can we use sid here?

Reply:
Yep. They don't seem to have any rolling release distros on there, and I couldn't find a spot on xpra.org that lists the most recent stable version besides this one.